manifests/0000_70_cluster-network-operator_02_rbac.yaml

```diff
@@ -1,16 +1,48 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cluster-network-operator-service-account
+  namespace: openshift-network-operator
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
```

Comment:
interesting! Can you add some pointers, either here or in the commit message with more info on these annotations?

Reply:
those magical annotations were mentioned https://github.com/openshift/enhancements/blob/master/enhancements/update/cluster-profiles.md#cluster-profile-annotation